pkg/manager/manager.go

```diff
@@ -50,6 +50,11 @@ type Manager interface {
 	// non-leaderelection mode (always running) or leader election mode (managed by leader election if enabled).
 	Add(Runnable) error
 
+	// Leading is closed when this manager becomes the leader of a group of
+	// managers, either because it won a leader election or because no leader
+	// election was configured.
+	Leading() <-chan struct{}
```

Comment:
non-blocking nit: is there a better name for this?  "closed when leading" strikes me as hella confusing

Reply:
Aside: the california slang has gotten to me :fearful:

Reply:
Haha, I started with `Elected()`, but @alvaroaleman convinced me `Leading` was more appropriate in https://github.com/kubernetes-sigs/controller-runtime/pull/863#discussion_r398860725. Any preference between those two?

The main detractor for `Elected` is that an election doesn't actually always happen, but "closed when elected" does seem more in line with similar examples ([like `ctx.Done()`](https://golang.org/pkg/context/#Context)), in that it reads "the channel is closed when the thing (election) finishes", rather than "the channel is closed when the thing (leading) starts".

Reply:
> I wanted something that showed/tested the behavior of Elected directly

I'm hoping from the comment below that you prefer `Elected`, which is also my preference, so I've pushed a commit that re-renames it.